proxy/coco/src/state.rs

```diff
@@ -90,23 +90,27 @@ where
 ///   * Fails to verify `User`.
 ///   * Fails to set the default `rad/self` for this `PeerApi`.
 #[allow(clippy::single_match_else)]
-pub async fn init_owner(peer: &Peer<BoxedSigner>, name: String) -> Result<LocalIdentity, Error> {
+pub async fn init_owner(
+    peer: &Peer<BoxedSigner>,
+    name: String,
+    ethereum: Option<EthereumClaimExtV1>,
```

Comment:
We should change this to just take `PersonPayload` and remove `name` and `ethereum`. Or even take `impl Into<PersonPayload>` so that the parameter can also easily be an `api` type.

Reply:
Done and moved payload extension to API